src/test/java/com/thebois/models/inventory/items/ItemTests.java

```diff
@@ -0,0 +1,27 @@
+package com.thebois.models.inventory.items;
+
+import java.util.stream.Stream;
+
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import static org.assertj.core.api.Assertions.*;
+
+public class ItemTests {
+
+    public static Stream<Arguments> getItemTypesAndItemClass() {
+        return Stream.of(
+            Arguments.of(ItemType.LOG, new Log()),
+            Arguments.of(ItemType.ROCK, new Rock()));
+    }
+
+    @ParameterizedTest
+    @MethodSource("getItemTypesAndItemClass")
+    public void fromTypeReturnsCorrectItemType(final ItemType itemType,
```

Comment:
style

There's no `fromType` here as far as I can see, it's `getType`.

Reply:
👍